src/errors.rs

```diff
@@ -145,17 +113,3 @@ pub enum Error {
 pub(crate) fn convert_bincode_error(err: bincode::Error) -> Error {
     Error::Serialisation(err.as_ref().to_string())
 }
-
-/// Entry error for `Error::InvalidEntryActions`.
-#[derive(Error, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
-pub enum EntryError {
```

Comment:
Why are we removing this?

Reply:
We're putting them in the main Error struct now. So there's only one error type. It didn;t make sense to me to have two... Not sure why we did that?

Reply:
It was specifically for the Sequence type's entry errors which were masked in `InvalidEntryActions` back then. But since we are moving towards a one main Error struct, I think removing this would make sense and add it to the main one.